util_test.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestMakeSecurityPolicyURL(t *testing.T) {
+	var tests = []struct {
+		name           string
+		projectID      string
+		region         string
+		securityPolicy string
+		expectedURL    string
+	}{
+		{
+			"non-empty securitypolicy",
```

Comment:
just a funny style thing but I usually see _either_ a one-liner where the field names are not specified, _or_ I see field names, I.e.

```
{ "non-empty securitypolicy", "my-project-id", ...}
```

or

```
{
  name: "non-empty securitypolicy"
  projectID: "my-project-id"
}
```

for table tests I tend to prefer the latter - it's verbose but :shrug:

Reply:
Ahh the example I saw didn't have the field names, and I was like "isn't it incredibly confusing"? lol